### DISCUSSION TITLE
unable to pull the image from a private repo from dockerhub

### DISCUSSION THREAD

**Discussion by venkatd7** [argoproj/argo-cd#13875] [Q&A]
Hi 

I'm unable to pull image from a private repo from dockerhub(able to pull from a public repo).

I created secret with following command 
kubectl create secret docker-registry argocd-docker-secret --docker-server="https://docker.io" --docker-username=abcdef --docker-password=abcdef --docker-email=abcdef@abcdef.ai -n argocd

I edited argocd-image-updater-config 
kubectl edit configmap argocd-image-updater-config -o yaml -n argocd

apiVersion: v1
data:
  log.level: debug
  registries.conf: |
    registries:
    - name: Docker Hub
      api_url: https://registry-1.docker.io
      prefix: docker.io
      ping: no
      credentials: pullsecret:argocd/argocd-docker-secret
kind: ConfigMap
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"v1","kind":"ConfigMap","metadata":{"annotations":{},"labels":{"app.kubernetes.io/name":"argocd-image-updater-config","app.kubernetes.io/part-of":"argocd-image-updater"},"name":"argocd-image-updater-config","namespace":"argocd"}}
  creationTimestamp: "2023-06-01T10:06:12Z"
  labels:
    app.kubernetes.io/name: argocd-image-updater-config
    app.kubernetes.io/part-of: argocd-image-updater
  name: argocd-image-updater-config
  namespace: argocd
  resourceVersion: "11275"
  uid: cb6880f3-ceaf-4fe0-b670-133b2dc37ce7

**Comment by 0x524c** [comment]
I'm facing the same issue.
